docs/glossary.md

```diff
@@ -36,7 +36,7 @@ FST
 A **finite-state transducer** is a computational formalism that is
```

Comment:
FST is a computational data-structure, which results from compiling with a FST-compiler plain-text source according to e.g. the LEXC of TWOLC of XFSCRIPT formalisms (if I've understood things correctly).